src/main/java/org/jabref/model/search/rules/SearchRules.java

```diff
@@ -25,6 +29,10 @@ public static SearchRule getSearchRuleByQuery(String query, boolean caseSensitiv
         }
     }
 
+    private static boolean isSimpleQuery(String query) {
+        return StringUtil.isBlank(query) || simpleExpression.matcher(query).matches();
```

Comment:
Do you really need both? The current regex has a star (`*`) at the end, which means zero (`0`) ore more matches. So, the empty string should match, too.

Reply:
Good suggestion, that sounds reasonable. I removed the StringUtil dependency and as far as I can see there is no difference in the search.